src/metrics.rs

```diff
@@ -306,6 +307,17 @@ impl<'a> FuncSpace<'a> {
         color!(stdout, White);
         writeln!(stdout, "{}", val)
     }
+
+    fn dump_json(&self, path: &PathBuf, output_path: &PathBuf) -> std::io::Result<()> {
+        let json_data = serde_json::to_string_pretty(&self).unwrap();
+
+        let mut file = path.as_path().file_name().unwrap().to_os_string();
+        file.push(".json");
+
+        let mut json_path = output_path.clone();
+        json_path.push(file);
+        write_file(&json_path, json_data.to_string().as_bytes())
```

Comment:
how to deal with an existing file ?
For example, you have `./foo/a.py` and `./bar/a.py`, the two files `a.py.json` will be the same.

Reply:
We could look if there is already an `a.py.json` file in the `output_path` directory and replace filenames with paths (e.g `./foo/a.py.json` and ./bar/a.py.json`) 

What do you think of this approach?

Reply:
It's ok for me, let's do that and if someone complains at some point we'll add an option or whatever...

Reply:
> It's ok for me, let's do that and if someone complains at some point we'll add an option or whatever...

I agree

> FYI, @mozilla we use the server part for now.

I see, that's why is full of tests, thanks for the info! :)

Reply:
@calixteman 
Done. I've replaced slashes, backslashes and colons with `_`.